crates/conformance-test-utils/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "conformance-test-utils"
+version = { workspace = true }
+edition = { workspace = true }
+description = { workspace = true }
+homepage = { workspace = true }
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+clap = { version = "4.2", features = [ "derive" ] }
+starknet = "0.4.0"
+
+[dev-dependencies.cargo-husky]
```

Comment:
do we need to define this for each crate?

Reply:
This seems like a better approach when dealing with workspaces for `cargo-husky` https://github.com/rhysd/cargo-husky/issues/18#issuecomment-1500989739, should we adopt this?